spmd/compiler/fusion.py

```diff
@@ -162,22 +214,448 @@ def _scan_graph_for_fusion_elements(
     return element_list
 
 
-def run_comm_fusion(gm: fx.GraphModule) -> bool:
+def _copy_fe_to_buffer(
+    gi: GraphInfo, gm: fx.GraphModule, in_fe_list: list[FusionElement]
+) -> None:
+    """first half of fusion - move desired items to buffer and create graph"""
+    buffer_node = gi.global_buffer_node
+    buffer_size = gi.global_buffer_size
+
+    copy_list = in_fe_list
```

Comment:
any reason for not using in_fe_list consistently?

Reply:
not overall. I was using the in_fe_list before splitting the function into two different ones and this was a rename to make the code clearer in the copy side.
I've removed in_fe_list and simply work with copy_list as the input param and stay consistent from there. 
Thanks for the spot!